README.md

```diff
@@ -143,15 +143,16 @@ as they can be built by running a Unix command in a container.
 Getting started
 ===============
 
-Docker can be installed on your local machine as well as servers - both
-bare metal and virtualized.  It is available as a binary on most modern
-Linux systems, or as a VM on Windows, Mac and other systems.
+The easiest way to get started with Docker in development is to use [Docker
+Toolbox](https://www.docker.com/docker-toolbox). It is an installer for Mac OS X
+and Windows that installs everything you need to build apps with Docker.
+
+To install on Linux or on servers, check out the [full installation instructions
```

Comment:
Toolbox is only intended for Mac and Windows, so I think this is misleading for users.  The audience for the README on GitHub is expected to be more technical not less. So, I think we should point them here:

http://docs.docker.com/engine/installation/